docs/MAVLINK_PARAMETERS.md

```diff
@@ -0,0 +1,32 @@
+# mLRS Documentation: MAVLink Parameters #
+
+([back to main page](../README.md))
+
+When enabled, the Tx module and receiver can also be configured via MAVLink parameters using a ground control station (GCS) such as MissionPlanner. This can be useful in cases in which the other methods for configuration, i.e., the CLI, Lua script, or OLED, are not (easily) accessible, such as in some stand-alone applications.
```

Comment:
For the second sentence, perhaps:

"Due to limitations of the MAVLink parameter system, this method is only recommended when other configuration methods (CLI, Lua script, or OLED) are not easily accessible."

Would suggest combining paragraph 1 and 3, which both begin with 'When enabled...'